website/public/js/services/guideServices.js

```diff
@@ -239,6 +239,7 @@ function($rootScope, User, $timeout, $state) {
     if (!updateFn) return; // only run after user has been wrapped
     watcher(); // deregister watcher
     if (window.env.IS_MOBILE) return; // Don't show tour immediately on mobile devices
+    if (User.user.items.gear.owned['weapon_warrior_0']) return;//If the user has the training sword, then don't show the intro tour
```

Comment:
@TheHollidayInn I think we should still have the check for the training sword, since it's possible to halt the tour, buy the sword and then resume the tour. It of course, would not be joined with the mobile check.

I want @lemoness's opinion on this, but wouldn't it be preferable to skip/change the rewards part of the tour if a sword has already been purchased, rather than canceling the whole tour?